### DISCUSSION TITLE
Holy DEPRECATION WARNING batman!

### DISCUSSION THREAD

**Discussion by kallisti5** [concourse/concourse#6379] [Help & Support]
I just applied our pipelines to the latest concourse and have about 1000+ DEPRECATION WARNING's.
So it seems we can't use _ (underscore) **or** space in names anymore?

Examples:
  - jobs.haiku-master-arm.plan.do[1].task(build master @minimum-mmc): 'build master @minimum-mmc' is not a valid identifier: illegal character ' '
  - jobs.haiku-master-x86_gcc2h: 'haiku-master-x86_gcc2h' is not a valid identifier: illegal character '_'
  - jobs.haiku-master-x86_gcc2h.plan.do[1].task(build master @nightly-anyboot): 'build master @nightly-anyboot' is not a valid identifier: illegal character ' '
  - jobs.haiku-master-sparc.plan.do[1].task(build master @minimum-raw): 'build master @minimum-raw' is not a valid identifier: illegal character ' '

Not being able to use a space *or* an underscore is extremely limiting for identifying things. As an example:

* build x86_64
* build x86_gcc2
* build x86

Dropping the spaces is easy enough:
* build-x86_64
* build-x86_gcc2
* build-x86

Dropping the spaces *and* underscores is painful though (especially since architecture names commonly have _'s in them.

**Comment by chenbh** [comment]
It will be allowed again in the next release (https://github.com/concourse/concourse/pull/6338), after feedback from the community at https://github.com/concourse/concourse/issues/6070#issuecomment-734518950
